apps/fluent-tester/src/FluentTester/TestComponents/Notification/NotificationTest.tsx

```diff
@@ -9,128 +10,464 @@ const svgProps: SvgIconProps = {
 };
 const iconProps = { svgSource: svgProps };
 
-const PrimaryTest: React.FunctionComponent = () => {
+const PrimaryTestWithAutoHide: React.FunctionComponent = () => {
+  const [visible, setVisible] = React.useState(false);
+  const onButtonPress = () => setVisible(!visible);
+
+  const [hidden, setHidden] = React.useState<boolean>(!visible);
+  const height = React.useRef(new Animated.Value(0)).current;
+
+  const show = () => {
+    setHidden(false);
+    Animated.sequence([
+      Animated.timing(height, {
+        toValue: -50,
+        duration: 600,
+        useNativeDriver: true,
+        easing: Easing.elastic(1.5),
+      }),
+      Animated.delay(3000),
+      Animated.timing(height, {
+        toValue: 0,
+        duration: 250,
+        useNativeDriver: true,
+      }),
+    ]).start(() => {
+      setHidden(true);
+      setVisible(false);
```

Comment:
Do we need both variables? Can't we just set one / simplify our logic so we only use one?

Reply:
The main reason we need both variables is because one controls the animation and the other controls whether or not to render the notification. If we remove the hidden variable, lines such as 73, 130 will be updated to {visible && ( ... )}. The minute the hide button is clicked, the visible variable will flip and the notification will disappear before you see the hide animation.